android/app/src/main/AndroidManifest.xml

```diff
@@ -29,7 +29,7 @@
       <activity android:name="com.facebook.react.devsupport.DevSettingsActivity" />
       <meta-data
       android:name="com.google.android.geo.API_KEY"
-      android:value="Your Google maps API Key Here"/>
+      android:value="AIzaSyDx4A76y-k3PaQ1C9LsCIR4kFkOUkqLxvI"/>
```

Comment:
We really need a better solution for this... :)

Reply:
Oh snap